src/backend/joanie/core/models/contracts.py

```diff
@@ -281,3 +281,13 @@ def get_abilities(self, user):
         return {
             "sign": can_sign,
         }
+
+    def is_fully_signed(self):
```

Comment:
IMO, we should use a `@property` decorator here.

Reply:
I agree with you.